.stylelintrc

```diff
@@ -6,6 +6,7 @@
     "stylelint-config-prettier"
   ],
   "rules": {
+    "no-missing-end-of-source-newline": 2,
```

Comment:
What? 😂 You just change the rules when they don't suit you?

Reply:
I enabled it.

Reply:
🆗